configuration/customize-dashboard.md

```diff
@@ -0,0 +1,38 @@
+---
+description:  This article describes how you can customize your Dashboard by rearranging the boxes. 
+---
+
+# Syskit Point Dashboard
+
+The **Syskit Point** Dashboard shows Syskit Point administrators the following information in boxes:
+ * Inventory
+ * Users
+ * Storage
+ * Security & Compliance
+ * External Collaboration
+ * Active Access Review
+ * Lifecycle Management
+ * Microsoft 365 Licenses
+
+The **Syskit Point** Dashboard shows Syskit Point collaborators the following information in boxes:
+ * Inventory
+ * Users
+ * Storage
+ * External Collaboration
+
+You can customize the order in which these feature boxes are shown based on which of the features shown on the dashboard are most important to you. 
```

Comment:
mislim da nigdje do sad nismo koristili pojam 'feature boxes'. 
ili ćemo koristiti tiles kao i do sad ili moramo provjeriti s Aleksandrom Lipovac kako to zvati